src/components/UserDropdownMenu/index.tsx

```diff
@@ -9,14 +9,13 @@ export const UserDropdownMenu = ({ handleOpenModal, setDropdownOpen }: UserDropd
   const { isAuth } = useAppSelector((state) => state.auth);
 
   return (
-    <div className={s.dropdownMenu}>
+    <div className={s.dropdownMenu} onClick={() => setDropdownOpen(false)}>
       {isAuth ? (
         <div>
           {userPanelButtonsList.map((button) => (
             <NavLink
               to={`account/${button.pathToPage}`}
               key={button.id}
-              onClick={() => setDropdownOpen(false)}
               className={({ isActive }) =>
               isActive
                 ? `cabinet-sidebar-nav_btn cabinet-sidebar-nav_active`
```

Comment:
Same situation, these style classes belong to the sidebar, here you should specify those that belong to the drop-down menu

Reply:
should i use @extend in global class, write own global class or copy them to local module.scss?

Reply:
You can write a global class like these, but with your own styles